solutions/fully-configurable/scripts/delete_secrets.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+API_KEY=$2
```

Comment:
Sensitive values should never passed to a script as an arg. They should be passed using environment variables.